src/scaffoldmaker/meshtypes/meshtype_3d_stellate1.py

```diff
@@ -367,8 +465,36 @@ def generateBaseMesh(cls, region, options):
                         element = mesh.createElement(elementIdentifier, elementtemplate1)
                         result = element.setNodesByIdentifier(eft1, nodeIdentifiers)
                         result3 = element.setScaleFactors(eft1, scalefactors) if scalefactors else None
+
+                        # add to meshGroup
+                        stellateMeshGroup.addElement(element)
+                        if isMouse:
+                            if na == 0:
+                                arm1MeshGroup.addElement(element)
+                            elif na == 1:
+                                arm2MeshGroup.addElement(element)
+                            elif na == 2:
+                                arm3MeshGroup.addElement(element)
```

Comment:
I think the arm and face groups could be defined for all stellates, not just mouse (the markers are just for mouse though).
Also, I'd prefer a more elegant implementation where you have an array of groups for arm index.
Try to avoid repeating code such as addElement; get the mesh groups by arm, then add the element with the same code for all cases.